bedrock/products/templates/products/vpn/variations/cta-b.html

```diff
@@ -0,0 +1,30 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "products/vpn/landing.html" %}
+
+ {% block hero_cta_change %}
+
+  {% set price = 'Get Mozilla VPN for ' + vpn_monthly_price(plan='12-month', lang=LANG) %}
+
+     {{ vpn_subscribe_link(
```

Comment:
The right price is being displayed depending on country, but the subscription ID is always being set to USD. To fix this you can do the following:

1. Add a div wrapper around this helper called `<div class="vpn-pricing-offer"></div>`.
2. Update the selector [geo.js](https://github.com/mozilla/bedrock/blob/master/media/js/products/vpn/geo.js#L204) uses to target this link to:

```javascript
var subscribeLinks = document.querySelectorAll('.vpn-pricing-variable-plans .js-fxa-product-button, .vpn-pricing-offer .js-fxa-product-button');
```

Reply:
Sorry, the link you need to update is this one https://github.com/mozilla/bedrock/blob/master/media/js/products/vpn/geo.js#L156